backend/station_manager/app/src/main/java/com/StationManager/app/services/handlers/commands/ResumeGeneratorCommandHandler.java

```diff
@@ -0,0 +1,4 @@
+package com.StationManager.app.services.handlers.commands;
+
+public class ResumeGeneratorCommandHandler {
```

Comment:
implement `CommandHandle<ResumeGeneratorCommand>` and just raise not implemented error inside the handle method

Reply:
Implemented